version/1/4/es/code_of_conduct.md

```diff
@@ -1,42 +1,42 @@
-# Código de Conducta convenido para Contribuyentes
+# Convenio de Código de Conducta para Contribuyentes
 
 ## Nuestro compromiso
 
-En el interés de fomentar una comunidad abierta y acogedora, nosotros como contribuyentes y administradores nos comprometemos a hacer de la participación en nuestro proyecto y nuestra comunidad una experiencia libre de acoso para todos, independientemente de la edad, dimensión corporal, discapacidad, etnia, identidad y expresión de género, nivel de experiencia, nacionalidad, apariencia física, raza, religión, identidad u orientación sexual. 
+Con el interés de fomentar una comunidad abierta y acogedora, nosotros como contribuyentes y administradores, nos comprometemos a hacer de la participación en nuestro proyecto y nuestra comunidad una experiencia libre de acoso para todos, independientemente de la edad, dimensión corporal, discapacidad, etnia, identidad y expresión de género, nivel de experiencia, nacionalidad, apariencia física, raza, religión, identidad u orientación sexual.
 
 ## Nuestros estándares
 
 Ejemplos de comportamiento que contribuyen a crear un ambiente positivo:
 
-* Uso de lenguaje amable e inclusivo
-* Respeto a diferentes puntos de vista y experiencias
-* Aceptación de críticas constructivas
-* Enfocarse en lo que es mejor para la comunidad
-* Mostrar empatía a otros miembros de la comunidad
+  * Uso de lenguaje amable e inclusivo.
+  * Respeto por los diferentes puntos de vista y experiencias.
+  * Aceptación de las críticas constructivas.
+  * Focalización en lo que es mejor para la comunidad.
+  * Mostrar empatía hacia los otros miembros de la comunidad.
 
-Ejemplos de comportamiento inaceptable por participantes:
+Ejemplos de comportamiento inaceptable por parte de los participantes:
 
-* Uso de lenguaje o imágenes sexuales y atención sexual no deseada
-* Comentarios insultantes o despectivos (*trolling*) y ataques personales o políticos
-* Acoso público o privado
-* Publicación de información privada de terceros sin su consentimiento, como direcciones físicas o electrónicas
-* Otros tipos de conducta que pudieran considerarse inapropiadas en un entorno profesional.
+  * El uso de lenguaje obsceno o imágenes sexuales y la atención sexual no deseada.
+  * "Trollear", insultar, comentarios despectivos, y ataques personales o políticos.
```

Comment:
I think you either use the comma or the conjunction y, but not both (after 'despectivos'). In this case, as it's the last item, I think you should omit the comma and just use y.